src/mod/applications/mod_conference/mod_conference.c

```diff
@@ -1343,6 +1345,38 @@ void conference_xlist(conference_obj_t *conference, switch_xml_t x_conference, i
 
 		switch_snprintf(tmp, sizeof(tmp), "%d", member->volume_out_level);
 		add_x_tag(x_member, "output-volume", tmp, toff++);
+
+		// relationships
+		x_relationships = switch_xml_add_child_d(x_member, "relationships", toff++);
+		switch_assert(x_relationships);
```

Comment:
Maybe there's one more thing. switch_assert ... I think we should check for NULL pointer (and optionally, whenever useful log it) instead of crashing FS? I know dozens of places use assert macro but these are mostly debugging leftovers - we should probably not be adding anymore of that?